### ISSUE TITLE
Version 1.1.0's get_field() implementation is backwards incompatible

### ISSUE THREAD

**Issue by jkirschner42** [KristianOellegaard/django-hvad#233]
The get_field function in django <=1.7 gets passed a many_to_many arguments, hence leading to an exception when used with hvad-1.1.0. 

To make it work SmartGetFields needs an many_to_many argument in the call function (but this might not work with django 1.8 where they removed the many_to_many argument)...

```python
class SmartGetField(object):
    ''' Smart get_field that raises a helpful exception on get_field() '''
    def __init__(self, real):
        assert not isinstance(real, SmartGetField)
        self.real = real

    def __call__(self, meta, name, many_to_many=True):
        try:
            return self.real(name)
        except FieldDoesNotExist as e:
            try:
                meta.translations_model._meta.get_field(name, many_to_many=many_to_many)
            except FieldDoesNotExist:
                raise e
            else:
                raise WrongManager("To access translated fields like %r from "
                                   "an untranslated model, you must use a "
                                   "translation aware manager. For non-translatable "
                                   "models, you can get one using "
                                   "hvad.utils.get_translation_aware_manager.\n"
                                   "For translatable models, use the language() "
                                   "method." % name)
´´´

**Comment by spectras** [comment]
Good catch. I'll publish a minor bugfix release today.

**Comment by spectras** [comment]
The issue should be fixed, and fix published in 1.1.1. Please tell me if it does not work as intended.
Thanks for reporting :)
